### PR TITLE
Fixed deletion of subscriptions

### PR DESCRIPTION
**Pull request by szagi3891**

### CODE REVIEW

File: crates/vertigo/src/computed/dependencies/graph.rs [hunk 1]
```diff
@@ -116,20 +107,22 @@ impl Graph {
         }
     }
 
-    pub fn get_edges_to_refresh(&self, edges: &BTreeSet<GraphId>) -> Vec<GraphValueRefresh> {
+    pub fn has_listeners(&self, parent_id: &GraphId) -> bool {
+        self.parent_childs.relation_len(&parent_id) > 0
+    }
 
+    pub fn get_for_dropped(&mut self) -> Vec<GraphId> {
```
Comment (sfisol) [inline crates/vertigo/src/computed/dependencies/graph.rs:114]: `drain_removables` would be a better suited name, as we drain the source empty

File: crates/vertigo/src/computed/dependencies/refresh_edges.rs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 use std::collections::{BTreeMap, BTreeSet};
 use crate::computed::{Dependencies, GraphId, graph_value::GraphValueRefresh};
 
-enum RefreshState {
+pub enum RefreshState {
     Unknown,
```
Comment (sfisol) [inline crates/vertigo/src/computed/dependencies/refresh_edges.rs:5]: `Pending` or `CalculationPending` would be a better name

**Review by sfisol — CHANGES_REQUESTED** [review]

File: crates/vertigo/src/computed/dependencies/refresh_edges.rs (hunk 2 above)
Comment (szagi3891) [reply crates/vertigo/src/computed/dependencies/refresh_edges.rs]: Done

File: crates/vertigo/src/computed/dependencies/graph.rs (hunk 1 above)
Comment (szagi3891) [reply crates/vertigo/src/computed/dependencies/graph.rs]: Done

**Review by sfisol — APPROVED** [review]

**Pull request merged**